x-pack/test/functional/apps/reporting_management/report_delete_pagination.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default ({ getPageObjects, getService }: FtrProviderContext) => {
+  const pageObjects = getPageObjects(['common', 'reporting']);
+  const log = getService('log');
+  const retry = getService('retry');
+  const security = getService('security');
+
+  const testSubjects = getService('testSubjects');
+  const esArchiver = getService('esArchiver');
+
+  describe('Delete reports', function() {
+    before(async () => {
+      await security.testUser.setRoles(['global_discover_read', 'reporting_user']);
+      await esArchiver.load('empty_kibana');
+      await esArchiver.load('reporting/archived_reports');
+      await pageObjects.common.navigateToActualUrl('kibana', '/management/kibana/reporting');
```

Comment:
Interesting. Cannot we use `common.navigateToApp('reporting')` here ? Just wondering if need to fix it

Reply:
yes, good suggestion.  Implemented it. 
There are bunch of other xpack apps that use navigateToActualURL() ...should change them to navigateToApp() in a separate PR.